packages/vm/benchmarks/bytecode.ts

```diff
@@ -0,0 +1,93 @@
+import { readFileSync } from 'fs'
+import Benchmark from 'benchmark'
+import { Chain, Common, Hardfork } from '@ethereumjs/common'
+import { getPreState } from './util'
+import { EVM } from '@ethereumjs/evm'
+import { Address, hexToBytes } from '@ethereumjs/util'
+import { DefaultStateManager } from '@ethereumjs/statemanager'
+
+class Stats {
+  runId?: number
+  iterationsCount!: number
+  totalTimeNs!: number
+  stdDevTimeNs!: number
+}
+
+type EvmCreator = () => Promise<EVM>
+
+async function runBenchmark(bytecode: string, createEvm: EvmCreator): Promise<Stats> {
+  const evm = await createEvm()
+
+  let promiseResolve: any
+  const resultPromise: Promise<Stats> = new Promise((resolve, reject) => {
+    promiseResolve = resolve
+  })
```

Comment:
```suggestion
  let promiseResolve: any
  const resultPromise: Promise<Stats> = new Promise((resolve, reject) => {
    promiseResolve = resolve
  })
  const {resolve: promiseResolve, promise: resultPromise} = Promise.withResolvers()
```
Should be able to do this

Reply:
I could be wrong, but I think that `Promise.withResolvers()` has been introduced in TS v5.4.0, and this repo's minimal requirement is `"typescript": "4.7.4"`. But I need someone with a better knowledge of this to confirm.

Reply:
I would just bump the typescript version up 4.7.4 is very old. Feel free to ignore though this is just a stylistic nit.